config/locales/devise/en.yml

```diff
@@ -116,7 +116,7 @@ en:
       max_login_attempts_reached: >
         Your account is temporarily locked because you have entered the
         one-time passcode incorrectly too many times.
-      please_try_again: Please try again in %{time_remaining}.
+      please_try_again: Please try again in <span id="countdown">%{time_remaining}</span>.
```

Comment:
Yeah I'm 100% ok with fixing that translation in the other file, just didn't want to distract you accidentally